ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -2235,6 +2311,16 @@ - (BOOL)isEqual:(id)object {
             (_step == castObject.step));
 }
 
+- (NSString*)stringForAnswer:(id)answer {
+    NSDate *date = [NSDate dateWithTimeIntervalSince1970:[answer doubleValue]];
+    NSCalendar *calendar = [NSCalendar currentCalendar];
+    calendar.timeZone = [NSTimeZone timeZoneWithName:@"UTC"];
+    NSDateComponents *components = [calendar components:(NSCalendarUnitHour | NSCalendarUnitMinute) fromDate:date];
+    NSDateComponentsFormatter *formatter = [[NSDateComponentsFormatter alloc] init];
+    formatter.unitsStyle = NSDateComponentsFormatterUnitsStyleFull;
+    return [formatter stringFromDateComponents:components];
```

Comment:
Have you tried `[ORKTimeIntervalLabelFormatter() stringFromTimeInterval:((NSNumber *)self.answer).floatValue];`?